SalesforceFoundation/EDA
# [EDA]: Update Course Connection's Backfill Message in EDA Setting

# Critical Changes

# Changes
## General Updates
- We've updated the message that displays when you run the Course Connection backfill to a toast message that aligns with other similar messages.

# Issues Closed
Closes  #1039

# New Metadata

# Deleted Metadata

# Testing Notes
1. Go to EDA Settings -> Course Connection tab -> Backfill tab
2. Check the box 'I understand and am ready to run Backfill' and the click the button "Run backfill"
3. A toast message should appear on the top with the text "The backfill was queued successfully."
